beetsplug/convert.py

```diff
@@ -204,24 +203,25 @@ def encode(self, command, source, dest, pretend=False):
         if not quiet and not pretend:
             self._log.info(u'Encoding {0}', util.displayable_path(source))
 
-        if not six.PY2:
-            command = command.decode(arg_encoding(), 'surrogateescape')
```

Comment:
This needs to stay (now unconditionally), right?

**EDIT:** I see you've added a decoding step in a second commit below, but now you're using `'utf8'` instead of `arg_encoding()`.

Reply:
Yep, this was a mistake! I think I made my life unintentionally difficult here by deleting this code when I should not have—I misread it as `if six.PY2`, i.e., the opposite sense, and then caused myself all manner of problems.